content/blog/posts/this-is-how-we-watchdog.md

```diff
@@ -0,0 +1,40 @@
+
+- 11/5 fatal shooting of Reginald Alexander Jr. in [Dallas](https://dpdbeat.com/2020/11/10/deputy-chief-reuben-ramirez-discusses-officer-involved-shooting-with-reginald-alexander-jr/); 
+- 11/18 fatal shooting of Pedro Martinez Jr. on the [Tyler Junior College](https://tylerpaper.com/news/local/breaking-tyler-man-dies-after-being-shot-by-tjc-officer/article_2fb73428-2a05-11eb-8221-6b9889e74ced.html) campus; 
+- 10/3 fatal shooting of Jonathan Price by [Wolfe City police](https://www.fox4news.com/news/hunt-county-grand-jury-reviewing-wolfe-city-officers-murder-case); and 
+- 10/9 fatal shooting of Ariel Esau Lujan by [Houston police](https://cityofhouston.news/investigation-into-fatal-shooting-at-6229-almeda-genoa-road/). 
+
+Then, I noted that there were *two* reports filed by separate people at the Texas Department of Criminal Justice for the 11/15 death of Omar Rojas: 
+
+![](https://res.cloudinary.com/texas-justice-initiative/image/upload/v1608508768/OmarRojas_OAGSite.png)
+
+And finally, the officer-involved shooting report for Reginald Alexander Jr. had been submitted by Dallas police, though media reported that he was shot by officers from another agency. 
+
+For each of these inconsistencies, I emailed the person at the agency responsible for filing the report and pointed out what I'd found. A couple immediately responded. One had misunderstood the law and filed the missing report that day, and another one said: 
+
+> Hello Ma'am
+> The officers involved do not work for the Dallas Police Department. 
+>
+> They are employed by the Dallas County community College system. They were unclear on whether or not they were set up to input information into the database. 
+>
+> I did not see a way to include in the drop down menu. 
+>
+> I will add the custodial death report because I do not beleive that (they) have done so. 
+
+*(I replied and advised that he run that by the OAG to be sure).* 
+
+My best hope is that the agencies amend/file the reports, but if they don't, my only recourse is to point out to the Office of the Attorney General that the reports are missing. Even then, the OAG doesn't really have to take any action – they are merely a repository for those reports. Not filing a custodial death report is a Class B misdemeanor, states a [law passed in 1983](https://www.kxan.com/dead-in-custody/?ipid=related-recirc), though the punishment has never been used. 
+
+My approach, although toothless, often works. This year, agencies have submitted 21 custodial death reports after I pointed out that the report was missing. But agencies can simply ignore my emails if they so choose, and then all I can do is tattle. It's great that Texas requires the collection of these reports, but watchdogging it can be a challenge, especially when the law has no teeth.
```

Comment:
How about emphasizing "21 custodial death reports" in bold? I think that's pretty substantial amount of reports that you made them file!